src/cephadm/cephadm.py

```diff
@@ -5558,6 +5588,14 @@ def _get_parser():
     parser_list_images = subparsers.add_parser(
         'list-images', help='list all the default images')
     parser_list_images.set_defaults(func=command_list_images)
+
+    parser_update_service = subparsers.add_parser(
+        'update-osd-service', help='update service for provided daemon')
+    parser_update_service.set_defaults(func=command_update_osd_service)
+    parser_update_service.add_argument('--fsid', help='cluster FSID')
+    parser_update_service.add_argument('--name', required=True, action=CustomValidation, help='Comma seperated daemon name (type.id1,type.id2)')
```

Comment:
given this command is specifically for updating OSDs, I think the help message should reflect that rather than the ambiguous `type`. I don't want people to start trying this on other daemon types as that could have some undesirable effects. In fact, I'm wondering if it would be better to make the argument called `--osd-ids` rather than name and we can just put `osd.` in front of the passed ids similar to the mgr module command

Reply:
Addressed in latest commit